engine/table/src/main/java/io/deephaven/engine/table/impl/PushdownResult.java

```diff
@@ -35,6 +41,23 @@ public class PushdownResult implements SafeCloseable {
      */
     public static final long DEFERRED_DATA_INDEX_COST = 50_000L;
 
+    private static final boolean IS_CI = "true".equalsIgnoreCase(System.getenv().get("CI"));
```

Comment:
Why don't we write specific tests that run in CI instead of annotating the production class?